src/pages/login/index.js

```diff
@@ -0,0 +1,55 @@
+import "./index.scss";
+import logo from "../../assets/logo.svg";
+import { TabForm } from "../../components/tabs";
+import { InputForm } from "../../components/input";
+import { Button } from "../../components/button";
+import { useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { FaCheck } from "react-icons/fa";
+import logoSignup from "../../assets/signup-vector-top.svg";
+export const LoginField = () => {
+  const [checked, setChecked] = useState(true);
+  console.log(checked);
```

Comment:
remove this console

Reply:
import { TabFrom, InputForm, Button } from "../../components"